changelogs/unreleased/8720-hu-keyu

```diff
@@ -0,0 +1 @@
+Log doesn't show pv name, but displays %!s(MISSING) instead
```

Comment:
Changelog filename needs to be with PR number not issue number -- so 8771-hu-keyu